pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -80,18 +104,19 @@ func (s *LDAPGroupSyncer) determineUsernames(members []*ldap.Entry) ([]string, e
 		if err != nil {
 			return nil, err
 		}
+		glog.V(2).Infof("Found OpenShift username %q for LDAP user for %v", username, member)
+
 		usernames = append(usernames, username)
 	}
 	return usernames, nil
 }
 
-// updateGroup finds or creates the OpenShift Group that needs to be updated, updates its' data, then
-// uses the GroupClient to update the Group record
-func (s *LDAPGroupSyncer) updateGroup(ldapGroupUID string, usernames []string) error {
+// makeOpenShiftGroup creates the OpenShift Group object that needs to be updated, updates its' data
+func (s *LDAPGroupSyncer) makeOpenShiftGroup(ldapGroupUID string, usernames []string) (*userapi.Group, error) {
```

Comment:
does this mean you can't sync an LDAP group to an existing group? you have to delete it and start from scratch?